dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/DataSourceController.java

```diff
@@ -339,7 +339,7 @@ public Result authedDatasource(@ApiIgnore @RequestAttribute(value = Constants.SE
     @AccessLogAnnotation(ignoreRequestArgs = "loginUser")
     public Result getKerberosStartupState(@ApiIgnore @RequestAttribute(value = Constants.SESSION_USER) User loginUser) {
         // if upload resource is HDFS and kerberos startup is true , else false
-        return success(Status.SUCCESS.getMsg(), CommonUtils.getKerberosStartupState());
+        return success(Status.SUCCESS.getMsg(), true);
```

Comment:
The parameter `principal` and `KerberosStartupState` are strongly correlated. I don't think this needs to be modified. We should add a description to the document to tell users this.

Reply:
This is a code that should not be submitted.